knotx-it-tests/src/test/resources/content/local/multiple-forms.html

```diff
@@ -43,7 +43,7 @@ <h2>
   </div>
```

Comment:
Can we limit the html file content?

Reply:
There are too many changes that are not connected with introducing template engine.
I've created #34 to do them.